### ISSUE TITLE
Sample App Crashing in a different way

### ISSUE THREAD

**Issue by KCErb** [infinitered/bluepotion#96]
So I changed the name to `TimeClock` and now everything goes fine until `rake newclear`. I get to the install and start steps (after many a fine compiler warning) and then the app crashes.

Since I wasn't sure about a possible device / genymotion conflict. I tried on both with `rake` and `rake device`. I followed all of the steps on the first page (gradle:install, API 16 etc) but at the end of the day this is what I get:

```
KC:TimeClock$ rake device
   Install ./build/Development-16/TimeClock.apk
5401 KB/s (7723878 bytes in 1.396s)
     Start com.your_company.TimeClock/.PMHomeActivity
--------- beginning of /dev/log/main
--------- beginning of /dev/log/system
  open fragment: #<HomeScreen:0x34d00051>
E/dalvikvm(23974): VM aborting
```

What to do ?

**Comment by skellock** [comment]
Not sure without looking at the code.

I'll tell you what though.  I've actually seen Genymotion get into some funky states.  Especially when I start changing the name of app or package name.

I have no explanation, but what I've found sometimes is to remove the Android app entirely from the emulator.

Give that a shot.

**Comment by skellock** [comment]
After a refreshing sleep...  I think this is related to #94 .  Are you running RM 4.0?

**Comment by KCErb** [comment]
> Not sure without looking at the code.

I should have been more clear: I don't have any custom code yet. This is crashing on just 
```
$ bluepotion create TimeClock
$ cd TimeClock
$ bundle
$ rake gradle:install
$ rake newclear
```

By crashing I mean that `rake newclear` `rake` and `rake device` (for the sake of touching nothing, not even the Rakefile) with my device or with Genymotion (both with no copy of `TimeClock` on them) all fail after Start, and none of them give a reason why. 

My device at least installs TimeClock, but on Genymotion I don't even see an icon indicating it installed (in spite of terminal output to the contrary).

> Especially when I start changing the name of app or package name.

Since I didn't have any code, I just created a new package from scratch rather than trying to change the app name.

> remove the Android app entirely

On my device I tried that to no avail, but in Genymotion it wasn't even getting an app onto the device!

> Are you running RM 4.0

:eyes:  yes
